components/selection/selection-observer-mixin.js

```diff
@@ -16,21 +16,21 @@ export const SelectionObserverMixin = superclass => class extends superclass {
 			 * @type {object}
 			 */
 			selectionInfo: { type: Object },
-			_provider: { type: Object, attribute: false }
+			_provider: { state: true }
 		};
 	}
 
 	constructor() {
 		super();
 		this.selectionInfo = new SelectionInfo();
-		this._provider = null;
+		this.__provider = null;
```

Comment:
I don't love this double-underscore for the private variable, it means the code now has `_provider` and `__provider` - I'm open if anyone thinks there's a better naming option.

Reply:
Yeah it's kinda awkward eh. We _could_ make the property public, but just configure it so it's undocumented. I have mixed feelings either way.